src/widgets/histogram/histogram-widget-model.js

```diff
@@ -53,6 +47,27 @@ module.exports = WidgetModel.extend({
   cancelAutoStyle: function () {
     this.dataviewModel.layer.restoreCartoCSS();
     this.set('autoStyle', false);
+  },
+
+  getState: function () {
+    var state = WidgetModel.prototype.getState.call(this);
+    var start = this.dataviewModel.get('start');
+    var end = this.dataviewModel.get('end');
+    var min = this.get('min');
+    var max = this.get('max');
+    var checkRoughEqual = function (a, b) {
+      if (_.isNumber(a) && _.isNumber(b) && Math.abs(start - min) > Number.EPSILON) {
```

Comment:
I though the problem was checking the actual state with the serialized one not the histogram range, shouldn't be this Math.abs(a - b) ?